docs/application/dotnet/guides/TOC.md

```diff
@@ -13,16 +13,26 @@
 ## [Package Manager](app-management/package-manager.md)
 
 # [Natural User Interface](nui/NUIoverview.md)
-## [NUI Hello World Tutorial](nui/hello-world.md)
+## [Setting Up the NUI Development Environment on Ubuntu](nui/setup-ubuntu.md)
+## [Getting Started](nui/hello-world.md)
+## [Layer](nui/layer.md)
 ## [UI Components](nui/ui-components.md)
+### [Styling UI Components](nui/styling-controls-with-JSON.md)
 ### [Visuals](nui/visuals.md)
-### [Button](nui/button.md)
+### [View](nui/view.md)
+### [Buttons](nui/button.md)
 ### [FlexContainer](nui/flexcontainer.md)
 ### [ImageView](nui/imageview.md)
+### [ScrollView](nui/scrollview.md)
+### [Slider](nui/slider.md)
+### [TableView](nui/tableview.md)
 ### [TextLabel](nui/textlabel.md)
-### [CustomView](nui/creating-custom-view-controls.md)
-### [Styling Components with JSON](nui/styling-controls-with-JSON.md)
+### [TextField](nui/textfield.md)
+### [TextEditor](nui/texteditor.md)
+### [VideoView](nui/videoview.md)
 ## [Animation](nui/animation.md)
```

Comment:
@Seoyeon2Kim Suggestion : I can see you separated animation-types.md out from animation.md. However I can see still many similar contents in two pages. Wouldn't it better to remove the separated out contents from animation.md?

Reply:
@mijin85cho  You're right. I'll remove AnimationTypes contents of animation.md

Reply:
@Seoyeon2Kim Your update in #160 is also applied. 
By the way, as you can see in animation.md of #189, LB has removed duplicated information between guide & api reference files. So the charts are removed, while you added a link to animation-types.md at one of them.
It looks like  animation-types.md is to describe the way of animation, and I'm agonizing where to add a link to it in animation.md. Would you give me a suggestion?

Reply:
@mijin85cho I took a look at the difference between the guide and API reference descriptions, but as you mentioned, you're right. They're duplicated.
So, I think you don't need to update the link in animation.md file. I prefer to remove the duplicated guide of animation.md, like #189 .
Should I delete it of #160 ?

Reply:
@Seoyeon2Kim I think you don't need to work for that on #160. As #189 is a PR to resolve the conflicts of #160, the fix of #189 is the last version to be applied (to my understanding). So the result will be same with work of #189.

Reply:
@mijin85cho Okay, great. Then please ignore the updated link of #160 . Thank you.